index.html

```diff
@@ -8,20 +8,26 @@
   <title>Portfolio</title>
 </head>
 <body>
-<section class="bg">
+<section class="bg" id="Protfolio">
   <header class="mobile-menu">
     <a href="#" class="logo">Welcome</a>
       <Nav id="menu">
       <img src="./Images/icon-menu.png" alt="Menu Icon">
       </Nav>
   </header>
-  <nav class="desktop-menu">
+  <!-- <div class="closeBtn-container">
+    <img class="closeBtn" src="./Images/Icon-close.svg" alt="close menu button">
+    </div> -->
```

Comment:
- [ ] commented-out codes should be removed from the codebase in order to make it clean and easy to understand 😃. This is among the coding best practices and as developers we should follow the convention 😃 👍